src/iedit.cpp

```diff
@@ -213,6 +213,42 @@ void iedit_disp_cybereyes_menu(struct descriptor_data *d)
   send_to_char(CH, "Set Options: ^c%s^n\r\nEnter options (0 to quit): ", buf1);
 }
 
+void iedit_disp_cyberarms_menu(struct descriptor_data *d)
+{
+  CLS(CH);
+  for (int y = 0; y < NUM_ARMSMODS; y += 2)
```

Comment:
Value name needs to match awake.h, I think it was `NUM_ARM_MODS` or something?

Reply:
Should be matched up now